### ISSUE TITLE
Multiple messages selection

### ISSUE THREAD

**Issue by 532910** [vector-im/riotX-android#1768]
Please add support to select multiple messages for later deletion, forwarding or copying.

**Comment by alittlebitofit** [comment]
no updates? :/

**Comment by piersonleo** [comment]
Yeah, having this feature would be nice. Perhaps can do it after selecting option from the bottomsheet, it'll trigger the selection mode?

**Comment by matthew747y** [comment]
Please add support to select multiple messages for later deletion, forwarding or copying.
